tests/lib/snaps/connections/meta/snap.yaml

```diff
@@ -0,0 +1,15 @@
+name: connections
```

Comment:
When we deifne a snap for a specific test then we store it in the test dir instead. Then the tools (pack-local/install-local) will find it. The good part of storing in the test dir is that it is automatically restored after the test, in this case if you edit the snap definition, it is required to reset the change because another test could use it after this one.

Reply:
Moved the snap to the test directory.